src/main/java/com/parsing/api/ExtractorLotInformation.java

```diff
@@ -0,0 +1,223 @@
+package com.parsing.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.parsing.model.LotId;
+import com.parsing.repository.LotIdRepository;
+import com.parsing.model.LotResult;
+import com.parsing.model.LotStatus;
+import com.parsing.model.Participant;
+import com.parsing.model.Status;
+import com.parsing.repository.LotResultRepository;
+import com.parsing.repository.ParticipantRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.client.RestTemplate;
+
+import java.math.BigDecimal;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class ExtractorLotInformation {
```

Comment:
Consider renaming to LotInformationExtractor, in methods we put verb to the first place

Reply:
*It's noun to first place.